doc/classes/Vector3.xml

```diff
@@ -85,7 +86,7 @@
 			<argument index="0" name="b" type="Vector3">
 			</argument>
 			<description>
-				Return the squared distance (distance minus the last square root) to b. Prefer this function over distance_to if you need to sort vectors or need the squared distance for some formula.
+				Returns the squared distance (distance minus the last square root) to b. Prefer this function over distance_to if you need to sort vectors or need the squared distance for some formula.
```

Comment:
- "minus the last square root"? That doesn't make sense. "Returns the squared distance to [code]b[/code]."